lamb/README.md

```diff
@@ -0,0 +1,99 @@
+![Lamb](https://github.com/scalar-labs/kelpie-test/workflows/Lamb/badge.svg)
+
+# Kelpie Lamb
+Kelpie Lamb is a Kelpie extension for benchmarking of Scalar DL contracts.
+
+## Build
+
+```console
+$ ./gradlew shadowJar
+```
+
+## How to run a test
+Before benchmarking, you first have to set up your Scalar DL environment.
+1. Make Lamb configuration file. See [lamb_config.toml](./lamb_config.toml) as a sample.
+2. Make a contract configuration file to specify your contract
+    - Refer to [Contract configuration](#Contract-configuration)
+    - [sample_contract_config.json](./sample_contract_config.json) is a sample configuration
+3. Make a variable configuration file to configure variables defined in the contract configuration file
+    - Refer to [Variable configuration](#Variable-configuration)
+    - [sample_variable_config.json](./sample_variable_config.json_config.json) is a sample configuration
+4. Run a test with Kelpie
+    ```console
+    $ ${KELPIE}/bin/kelpie --config <LAMB_CONFIG>
+    ```
+
+### With docker
+#### Docker build
+```console
+$ ./gradlew docker
+```
+
+#### Run a test
+```console
+$ ./docker_run.sh -l <LAMB_CONFIG> -c <CONTRACT_CONFIG> -v <VARIABLE_CONFIG> -p <POPULATION_CONTRACT> -t <TARGET_CONTRACT>
+```
+- Your contract classpaths in the contract configuration file should be under `/lamb` directory because the script places your contracts to `/lamb` directory on docker, as seen below, for example.
+    ```json
+    "sample_contract.Balance": {
+      "class_file": "/lamb/Balance.class",
+      "properties": {},
+      "arguments": {
+        "asset_id": "_id_"
+      }
+    }
+    ```
+- You don't have to change the paths like the contract configuration file and the variable configuration file in the default [lamb_config.toml](./lamb_config.toml)
```

Comment:
```suggestion
- You don't have to change the paths of the contract configuration file and variable configuration file in the default [lamb_config.toml](./lamb_config.toml)
```
If they are parts of all,  I think `such as` instead of `of` is more appropriate.